### PR TITLE
feat: implement delete address entries in favourites

### PR DESCRIPTION
**Pull request by stefandunca**
Required for implementing favourite addresses in status-desktop [#6546](https://github.com/status-im/status-desktop/pull/7087)

### CODE REVIEW

**Review by richard-ramos — APPROVED** [review]

File: services/wallet/favourite.go [hunk 1]
```diff
@@ -44,3 +44,8 @@ func (fm *FavouriteManager) AddFavourite(favourite Favourite) error {
 	_, err = insert.Exec(favourite.Address, favourite.Name)
 	return err
 }
+
+func (fm *FavouriteManager) DeleteFavourite(address common.Address) error {
```
Comment (cammellos) [inline services/wallet/favourite.go:48]: Worth soft deleting? 
Otherwise it's difficult to sync them

Comment (Samyoul) [reply services/wallet/favourite.go:48]: Yes. and maybe also consider triggering sync of wallet in response to deletion.

Comment (stefandunca) [reply services/wallet/favourite.go:48]: @cammellos not sure what you mean by this. To mark them as hidden in DB? Won't this generate as a side effect having a hiden history of favorites?

**Comment by stefandunca** [comment]
dropped in favor of merging "favuorites" with "saved addresses" and keep favourite state PR here https://github.com/status-im/status-go/pull/2844

**Pull request closed without merging**